packages/amplify-category-api/src/provider-utils/awscloudformation/utils/check-appsync-api-migration.ts

```diff
@@ -0,0 +1,27 @@
+import { $TSContext, AmplifyCategories, getMigrateResourceMessageForOverride } from 'amplify-cli-core';
+import { printer, prompter } from 'amplify-prompts';
+import { AppsyncApiInputState } from '../api-input-manager/appsync-api-input-state';
+import { migrateResourceToSupportOverride } from './migrate-api-override-resource';
+
+export const checkAppsyncApiResourceMigration = async (context: $TSContext, apiName: string, isUpdate: boolean): Promise<boolean> => {
+  const cliState = new AppsyncApiInputState(apiName);
+  /**
+   * Below steps checks for TransformerV1 app and updates the FF { useexperimentalpipelinedtransformer , transformerversion}
```

Comment:
does this update the FF or just read it?

Reply:
updates the `transformerVersion` FF to 2  if `experimentalpipelinedTransformer` ff is true